packages/@aws-cdk/aws-ec2/lib/port.ts

```diff
@@ -9,6 +9,8 @@ export enum Protocol {
   UDP = 'udp',
   ICMP = 'icmp',
   ICMPV6 = '58',
+  ESP = 'esp',
+  AH = 'ah',
```

Comment:
This needs to be `ESP = ‘50’` and  `AH = ‘51’`

Reply:
Also there are many other protocols that need to be added to this enum
https://en.wikipedia.org/wiki/List_of_IP_protocol_numbers